FAIR_Training_LDP/.htaccess

```diff
@@ -1,4 +1,4 @@
 Options +FollowSymLinks
 RewriteEngine on
-RewriteRule ^(.*) http://ldp.cbgp.upm.es:8890/$1 [R=307,L]
+RewriteRule ^FAIR_Training_LDP/(.*) http://ldp.cbgp.upm.es:8890/$1 [R=307,L]
```

Comment:
Just checking, you want urls to be like https://w3id.org/FAIR_Training_LDP/FAIR_Training_LDP/...?  I ask because the PR title makes it sound like you meant to add that to your site url.

Reply:
ping

Reply:
I want the URLs to be  https://w3id.org/FAIR_Training_LDP/X/Y/Z and to redirect to http://ldp.cbgp.upm.es:8890/X/Y/Z

I'm trying to eliminate the FAIR_Training_LDP component from the redirect URL

Reply:
(sorry, I am logged in on my company account right now)

Reply:
I think that's what it was already doing?
```
$ curl -I https://w3id.org/FAIR_Training_LDP/X/Y/Z
HTTP/1.1 307 Temporary Redirect
Date: Thu, 08 Oct 2020 17:05:59 GMT
Server: Apache/2.4.29 (Ubuntu)
Access-Control-Allow-Origin: *
Location: http://ldp.cbgp.upm.es:8890/X/Y/Z
Content-Type: text/html; charset=iso-8859-1
```